src/test/java/seedu/address/logic/commands/DisplayCalendarCommandTest.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for ListCommand.
+ */
+public class DisplayCalendarCommandTest {
+/*
+TODO: implement this class
+
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(new AddressBook(), new UserPrefs(), getTypicalEdulogCalendar());
+        expectedModel = new ModelManager(new AddressBook(), new UserPrefs(), model.getEdulogCalendar());
+    }
+
+    @Test
+    public void execute_listIsNotFiltered_showsSameList() {
+        assertCommandSuccess(new ListCommand(), model, ListCommand.MESSAGE_SUCCESS, expectedModel);
```

Comment:
I like how you used the assert functions already implemented in AB3, increasing code reuses.